### DISCUSSION TITLE
Data Camp 3 Months Free Trial with GitHub Student Pack

### DISCUSSION THREAD

**Discussion by Barakataudu** [community/community#30071] [GitHub Education] [answered]
Hello, 

I tried getting the benefits of data camp free trials, but I keep getting an error message. What could be the issue? 
This is the error I keep seeing:

![image](https://user-images.githubusercontent.com/102633950/185613116-c2649fb6-4701-4ab6-8065-9b214ca5a2b5.png)

Also, it seems Canva Pro is no longer included in the pack? Will that change soon or it's a permanent decision?

Kind regards,

**Comment by Zoltus** [comment, marked as answer]
> Hello,
> 
> I tried getting the benefits of data camp free trials, but I keep getting an error message. What could be the issue? This is the error I keep seeing:
> 
> ![image](https://user-images.githubusercontent.com/102633950/185613116-c2649fb6-4701-4ab6-8065-9b214ca5a2b5.png)
> 
> Also, it seems Canva Pro is no longer included in the pack? Will that change soon or it's a permanent decision?
> 
> Kind regards,

Hi i got response from the data camp support team!

"Hey Zoltus,
 
Thank you for reaching out to DataCamp Support! My name is Emily, and I would be happy to help out.
 
Please be advised that we are re-evaluating the way the GitHub trial integrates with DataCamp. For that reason when trying to activate the trial plan you might be experiencing issues. We will review your account status and evaluate your eligibility manually.
 
You can review our Github FAQ here: https://www.datacamp.com/github-students. This offer is only available to students who are not already DataCamp subscribers. That said, this offer is applied to new free users only.
 
Could you please take a screenshot of the approval/dashboard of your Github student account?"

**Reply by Barakataudu** [reply]
Hi Zoltus, thanks a lot for your reply. I tried contacting the support team yesterday, but they haven't reached back to me. I have a free datacamp account, so not sure why I still couldn't get access.

**Reply by Zoltus** [reply]
> Hi Zoltus, thanks a lot for your reply. I tried contacting the support team yesterday, but they haven't reached back to me. I have a free datacamp account, so not sure why I still couldn't get access.

Most likely they will ask you to do same as i did. good luck:3

**Reply by Barakataudu** [reply]
Yes sure, thank you for answering

**Reply by DIDIHBABS** [reply]
i am having the same problem too !

**Reply by Zoltus** [reply]
> i am having the same problem too !

Send them support ticket containing proof that you are github student. 
For example this link https://education.github.com/discount_requests/student_application
should show on the right that you have been approved to student.
Also tellthem your datacamp account email they should manually add it

**Comment by MuhammadDaud01** [comment]
Hello!

I am not able to see the datacamp paid courses as part of 3 months plan. 
P.S I am verified as a student at Github and I tried to integrate github with datacamp.

**Reply by samus-aran** [reply]
Hi @MuhammadDaud01! 
The best place to go for assistance with your Global Campus Benefits like this one is to reach us by opening a ticket on our [Support page](https://support.github.com/contact/education?tags=hubberfy_education). Please select the "**I need help with a student developer pack offer**" option to receive the most appropriate team and provide as much detail as possible in your ticket (screenshots would be helpful). 👍

**Comment by DAlons27** [comment]
No se si sea de ayuda mi comentario, pero de tanto dar click en "Start Free Trial" en un lapso de 1 hora y media me dejo registrarme sin enviar nada a soporte.
